### PR TITLE
Issues/2

### LINKED ISSUE #2: Attendee Phone Number not displayed for editing.
When the ticket's **Edit Information** button is clicked, the **_Phone Number_** is not displayed.

### PR DESCRIPTION
**Pull request by ravinderk**
This PR will resolve #2

### CODE REVIEW

File: classes/class-camptix-payment-method-instamojo.php [hunk 1]
```diff
@@ -240,162 +255,164 @@ function payment_notify() {
...
 
-			curl_setopt($ch, CURLOPT_POST, true);
-			curl_setopt($ch, CURLOPT_POSTFIELDS, http_build_query($payload));
-			$response = curl_exec($ch);
-			curl_close($ch); 
 
-			$json_decode = json_decode($response , true);
-			
-			if($json_decode['success']){
+		$ch = curl_init();
```
Comment (vachan) [inline classes/class-camptix-payment-method-instamojo.php]: @ravinderk & @sanyogs 
We must use `wp_remote_post`
see: https://codex.wordpress.org/Function_Reference/wp_remote_post

Comment (ravinderk) [inline classes/class-camptix-payment-method-instamojo.php]: @vachan I am not working on instamojo, so can you create another issue for instamojo and let @sanyogs handle this.

Comment (vachan) [inline classes/class-camptix-payment-method-instamojo.php]: I have mentioned @sanyogs we can wait for him to replay, presently there is a branch conflict, will review and merge; @sanyogs can work on the instamojo part of it. Or if I get the time before Tue, 25-Jul-2017; I will git it a try if not done.

Comment (ravinderk) [inline classes/class-camptix-payment-method-instamojo.php]: @vachan can we merge this if it is working fine?
I created issue for @sanyogs  https://github.com/wpindiaorg/camptix-indian-payments/issues/8

and conflicts resolved.

**Pull request merged**